### PR TITLE
added config-file as parameter

### PR DESCRIPTION
**Pull request by xinredhat**

### CODE REVIEW

File: common/tasks/linters/yamllint/yaml-lint.yaml [hunk 1]
```diff
@@ -15,6 +15,10 @@ spec:
     - name: git-revision
       description: The Git revision (commit SHA) from which the test pipeline is originating.
       type: string
+    - name: config-file
```
Comment (flacatus) [inline common/tasks/linters/yamllint/yaml-lint.yaml:18]: remote-config-file

File: common/tasks/linters/yamllint/yaml-lint.yaml [hunk 2]
```diff
@@ -15,6 +15,10 @@ spec:
...
+      description: The configuration file for the YAML linter.
```
Comment (flacatus) [inline common/tasks/linters/yamllint/yaml-lint.yaml:19]: The path to the YAML linting configuration file located in the Git repository. 
        This configuration file specifies the rules and patterns for linting YAML files.
        It should be relative to the root of the repository.

File: common/tasks/linters/yamllint/yaml-lint.yaml (hunk 1 above)
Comment (xinredhat) [reply common/tasks/linters/yamllint/yaml-lint.yaml:18]: why remote-config-file?
the step `clone-refs` will first clone  repo, in generally the config-file should be defined in product repo. they would like to use their own

Comment (xinredhat) [reply common/tasks/linters/yamllint/yaml-lint.yaml]: fixed

File: common/tasks/linters/yamllint/yaml-lint.yaml (hunk 2 above)
Comment (xinredhat) [reply common/tasks/linters/yamllint/yaml-lint.yaml]: fixed

**Pull request merged**